uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Form/AbstractDrupalInteractionTaskForm.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+use CRM_HRCore_Service_DrupalUserService as DrupalUserService;
+use CRM_Utils_Array as ArrayHelper;
+
+abstract class CRM_HRCore_Form_AbstractDrupalInteractionTaskForm extends CRM_Contact_Form_Task {
```

Comment:
they class doesn't have any abstract method , it is doesn't make to make it abstract .. maybe : 

**class CRM_HRCore_Form_BaseDrupalInteractionTaskForm**

is better